frontend/src/components/Common/Pagination/index.tsx

```
@@ -0,0 +1,84 @@
+            onClick={handlePreviousPage}
+            className={currentPage === 1 ? 'cursor-not-allowed' : 'cursor-pointer'}
+          >
+            前へ
+          </PaginationItem>
+        )}
+
+        {paginateData.links.map((link, index) => {
+          const pageNumber = Number(link.label);
+
+          if (isNaN(pageNumber)) {
+            return null;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)